apps/coscrad-frontend/src/utils/test-utils/test-data/build-mock-index-response.ts

```diff
@@ -0,0 +1,16 @@
+import { IBaseViewModel, ICommandFormAndLabels, IIndexQueryResult } from '@coscrad/api-interfaces';
+
+export const buildMockIndexResponse = <T extends IBaseViewModel>(
```

Comment:
Nice helper. It's good that we have this single util to update if we tweak the API.